packages/core/src/components/BreadcrumbsBar/BreadcrumbItem/__stories__/breadcrumbItem.stories.tsx

```diff
@@ -52,13 +52,13 @@ export const States = {
       </div>
       <div className="monday-storybook-breadcrumb-item_row-wrapper">
         <span>Disabled</span>
-        <BreadcrumbsBar>
+        <BreadcrumbsBar type={BreadcrumbsBar.types.INDICATION}>
```

Comment:
There's a difference between hoe we expect users to use our components to how they're documented on our Storybook. This PR really exposed it 👏🏻